base/asyncevent.jl

```diff
@@ -90,6 +90,8 @@ mutable struct Timer
     cond::ThreadSynchronizer
     @atomic isopen::Bool
     @atomic set::Bool
+    timeout::Real
```

Comment:
I think we should store this as the computed UInt64 milliseconds. It can be printed in seconds with 3 decimals if you like. Not really worth introducing a type instability for this.